File: CP COURSE/promblems/D_Range_Sum.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
int t;
cin>>t;
while(t--){
    int a,b;
    cin>>a>>b;
    int sum1=(a)*(a+1)/2;
    int sum2=(b)*(b+1)/2;
    cout<<sum2-sum1+a<<endl;
}
return 0;
}

File: CP COURSE/promblems/D_Range_Sum.py
t=int(input())
for i in range(t):
    a,b=map(int,input().split())
    sum1=(a*(a+1))//2;
    sum2=(b*(b+1))//2;
    if(b>a):
        print(sum2-sum1+a)
    else:
        print(-(sum2-sum1+b))    